src/templates/rule.js

```diff
@@ -63,6 +63,22 @@ export default ({ data }) => {
 					</ul>
 					<hr />
 					{/* Description */}
+					<h2 id="description">
```

Comment:
**Note for reviewers:** I intend to abstract this code so nice clickable anchors can be easily added in various places.